clash-prelude/src/Clash/Class/AutoReg/Instances.hs

```diff
@@ -6,6 +6,7 @@
 {-# LANGUAGE TemplateHaskell #-}
 
 {-# OPTIONS_GHC -Wno-orphans #-}
+-- {-# OPTIONS_GHC -ddump-splices #-}
```

Comment:
Should this be included in the commit?

Reply:
Oh yes, I like having that in files with template haskell splices. It makes it a bit easier to debug!